src/test/scala/biowdl/test/QCBothReads.scala

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Maybe we can merge some files together?

Reply:
Every test a single file is quite nice. Are you referring to package objects?